platform_versioned_docs/version-24.1/enterprise/data-studios.mdx

```diff
@@ -31,6 +31,19 @@ You can configure which organizational workspaces have access to Data Studios. T
 - `allowed-workspaces: [ <WORKSPACE_ID>,<WORKSPACE_ID> ]`: Enables Data Studios for the comma-separated list of organizational workspace IDs.
 - `allowed-workspaces: null`: Enables Data Studios for all organizational workspaces.
 
+## Available data studio environment images
+
+Each of the provided Data Studios environments includes a particular version of the underlying data analytics software package.
+
+To quickly identify which version of the software an image includes, the version string for each container is in the form of `<software_version>-<seqera_version>`. For example, if the version string for the RStudio Server is `4.4.1-0.7.1`, version `4.4.1` is the RStudio version and `0.7.1` is the version of this Seqera-built container image.
+
+The latest environment versions are listed below:
+
+- JupyterLab: `public.cr.seqera.io/platform/data-studio-jupyter:4.1.5-0.7.1`
+- RStudio Server: `public.cr.seqera.io/platform/data-studio-rstudio:4.4.1-0.7.1`
```

Comment:
Just so you know, we also have an old RStudio version 4.0.0. @jimmypoms Do we want to list all of them here?

Reply:
Not sure :thinking: @seqeralabs/scientific-development is there value in having both rstudio 4.0 and rstudio 4.4 as data studio templates?

Another small thing: please use versions `major.minor` for the data studio client throughout the documentation (e.g. `public.cr.seqera.io/platform/data-studio-rstudio:4.4.1-0.7`). It makes our lives easier :)

Reply:
I think if we want these to be at all reproducible for people it's hard to justify deleting prior images. Though the list could get long over subsequent updates, so I'm not sure of the right path.

Reply:
@pinin4fjords, so for this, we've wanted to start exploring using the release notes for listing new images as they come out; they challenge is right now, we don't have an async release note set yet.

cc @justinegeffen @llewellyn-sl